### PR TITLE
chore: bump jackson-databind version

### PR DESCRIPTION
**Pull request by lct45**
### Description 
There's a bug in jackson-databind before 2.13.0, see details [here](https://nvd.nist.gov/vuln/detail/CVE-2020-36518), so we're bumping the version
### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-rest-model/pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+            <version>2.13.2.2</version>
```
Comment (lct45) [inline ksqldb-rest-model/pom.xml:53]: I considered using `${jackson.version}` here which we use in a few other poms like the `ksqldb-api-client` but those are for the group `com.fasterxml.jackson.datatype` so I wasn't sure if we wanted these two groups to always have to have the same version, although its possible that implicitly this happened before I added the version field

Comment (agavra) [reply ksqldb-rest-model/pom.xml:53]: can we introduce a property in the parent pom for this version? (e.g. `jackson.databind.version`)

Comment (lct45) [reply ksqldb-rest-model/pom.xml:53]: @agavra is that the proper way to do it instead of using the variable from `common`? I can do that

Comment (agavra) [reply ksqldb-rest-model/pom.xml:53]: defining a new variable gives us the flexibility to have different versions from `jackson.version`. Idk if that's necessary or not

**Comment by lct45** [comment]
@agavra for any branches before master that need to be fixes I can just cherry-pick this commit and push, is that what you've done in the past?

File: ksqldb-rest-model/pom.xml (hunk 1 above)
Comment (lct45) [reply ksqldb-rest-model/pom.xml]: Cool, thanks! Just updated swapped to a variable in parent pom

**Comment by agavra** [comment]
@lct45 - generally I target the earliest branch and then pint merge forward, but cherry-picking to an earlier branch and then pint merging works as well

**Review by agavra — APPROVED** [review]

**Comment by lct45** [comment]
With the version bump in common, we don't need to make this change anymore https://github.com/confluentinc/common/pull/446

**Pull request closed without merging**